request_a_govuk_domain/request/templates/start.html

```diff
@@ -3,6 +3,20 @@
 
 {% block title %}Get approval to use a .gov.uk domain name - GOV.UK{% endblock %}
 
+{% set message_3_html %}
+    <p class="govuk-body">Message 3 content</p>
+{% endset %}
+{% gds_cookie_banner %}
+    {% gds_cookie_banner_message headingText="Message 1 heading" %}
```

Comment:
is "Message 1 heading" intended?

Reply:
Thanks. The whole block is a left-over. Will remove